.github/workflows/ci.yml

```diff
@@ -3,6 +3,8 @@ name: v8-cmake
 on:
   pull_request:
     paths:
+      - 'CMakeLists.txt'
```

Comment:
Any thought on remove `paths`, it's okay we do build on every pr.

Reply:
I felt that was kind of wasteful when making changes to just the README or, say, update_v8.py. I try hard to reduce my carbon footprint and not running the CI unnecessarily is (a small) part of that. :-)